protocol/messenger.go

```diff
@@ -2790,7 +2784,26 @@ func (m *Messenger) SyncDevices(ctx context.Context, ensName, photoPath string,
 	return nil
 }
 
-func (m *Messenger) syncContactRequestDecision(ctx context.Context, requestID string, accepted bool, rawMessageHandler RawMessageHandler) error {
+func (m *Messenger) syncLatestContactRequest(ctx context.Context, rawMessageHandler RawMessageHandler) error {
```

Comment:
If I understood correctly, the name should be on the plural `syncLatestContactRequests` because we're syncing potentially more than one CR.